packages/browser/src/Session.ts

```diff
@@ -57,22 +57,17 @@ async function silentlyAuthenticate(
   clientAuthn: ClientAuthentication
 ) {
   // Check if we have an ID Token in storage - if we do then we may be
-  // currently logged in, and the user has refreshed their browser page. In
-  // this case, it can be really useful to save the user's current browser
-  // location, so that we can restore that location after we complete the
-  // entire re-login-the-user-silently-flow (e.g., if the user was on a
-  // specific 'page' of a Single Page App, then presumably they'll expect to
-  // be brought back to exactly that same 'page' after the full refresh).
-  const storedSessionInfo = await clientAuthn.getSessionInfo(sessionId);
-  if (
-    storedSessionInfo === undefined ||
-    storedSessionInfo.idToken === undefined
-  ) {
-    return;
-  }
-  window.localStorage.setItem(KEY_CURRENT_URL, window.location.href);
+  // currently logged in, and the user has refreshed their browser page. The ID
+  // token is validated, and its issuer claim is extracted to know where silent
+  // authentication is possible.
```

Comment:
It's not immediately clear here that the ID token is actually being validated - so should we rename `getCurrentIssuer()` to be `validateCurrentIssuer()` or something, just to try and make it clearer what's going on?
Slight typo here:
```suggestion
  // currently logged in, and the user has refreshed their browser page. The ID
  // token is validated, and its issuer claim is extracted to know if silent
  // authentication is possible.
```

Reply:
This isn't a type (although I guess the phrasing isn't great then ^^): the issuer claim indicates against which IdP silent authentication can be performed, hence the `where`

Reply:
37d12534b06a1eac66de53e6948c3e306bebea26 renames the function